src/components/AccountDrawer/AccountDrawerDisplay.test.tsx

```diff
@@ -30,255 +30,155 @@ import {
-    render(<AccountCard account={selectedAccount} />);
-    const tokenTiles = screen.getAllByTestId("token-tile");
-    await waitFor(() => {
-      expect(tokenTiles).toHaveLength(4);
+      const tzktLink = within(screen.getByTestId("asset-panel-tablist")).getByRole("link", {
+        name: "View on Tzkt",
+      });
+      const expectedLink = "https://ghostnet.tzkt.io/" + selectedAccount.address.pkh;
+      expect(tzktLink).toHaveProperty("href", expectedLink);
     });
 
-    {
-      const { getByTestId } = within(tokenTiles[0]);
-      expect(getByTestId("token-name")).toHaveTextContent("Hedgehoge");
-      expect(getByTestId("token-balance")).toHaveTextContent("10,000.000000");
-    }
-
-    {
-      const { getByTestId } = within(tokenTiles[1]);
-      expect(getByTestId("token-name")).toHaveTextContent("tzBTC");
-      expect(getByTestId("token-balance")).toHaveTextContent("0.00002205");
-    }
-
-    {
-      const { getByTestId } = within(tokenTiles[2]);
-      expect(getByTestId("token-name")).toHaveTextContent("FA1.2 token");
-      expect(getByTestId("token-balance")).toHaveTextContent("123");
-    }
-
-    {
-      const { getByTestId } = within(tokenTiles[3]);
-      expect(getByTestId("token-name")).toHaveTextContent("youves uUSD");
-      expect(getByTestId("token-balance")).toHaveTextContent("0.01921875");
-    }
-  });
+    it("is updated on network change", async () => {
+      render(<AccountCard account={selectedAccount} />);
+      await waitFor(() => expect(screen.getByTestId("asset-panel-tablist")).toBeVisible());
 
-  it("should display nfts under nfts tab", async () => {
-    const user = userEvent.setup();
-    render(<AccountCard account={selectedAccount} />);
+      {
+        await act(() => store.dispatch(networksActions.setCurrent(MAINNET)));
+        expect(screen.getByTestId("asset-panel-tablist")).toBeVisible();
+      }
 
-    await waitFor(() => {
-      expect(screen.getByTestId("account-card-nfts-tab")).toBeInTheDocument();
+      const tzktLink = within(screen.getByTestId("asset-panel-tablist")).getByRole("link", {
+        name: "View on Tzkt",
+      });
+      const expectedLink = "https://tzkt.io/" + selectedAccount.address.pkh;
+      expect(tzktLink).toHaveProperty("href", expectedLink);
     });
-
-    await act(() => user.click(screen.getByTestId("account-card-nfts-tab")));
-
-    expect(screen.queryAllByTestId("account-card-nfts-tab")).toHaveLength(1);
   });
 
-  it("displays accounts operations under operations tab if account has operations", async () => {
-    render(<AccountCard account={selectedAccount} />);
+  describe("Operations tab", () => {
+    beforeEach(() => {
+      jest.mocked(getCombinedOperations).mockResolvedValue([
+        {
+          ...mockTzktTezTransfer(
+            selectedAccount.address.pkh,
+            mockImplicitAccount(1).address.pkh,
+            1000000
+          ),
+          id: 1,
+        } as TzktCombinedOperation,
```

Comment:
just curious if you need this cast?

Reply:
Yep, but I changed it to a cast for the list instead.